indexd/index/drivers/alchemy.py

```diff
@@ -620,6 +684,26 @@ def migrate_3(session, **kwargs):
         "CREATE INDEX {tb}__file_name_idx ON {tb} ( file_name )"
         .format(tb=IndexRecord.__tablename__))
 
+
+def migrate_4(session, **kwargs):
```

Comment:
you don't need migration scripts for creating new table.
Base.create_all() will detect missing tables and create them

Reply:
thanks, removed it